### PR TITLE
SDPA-436 SDPA-437 SDPA-438 SDPA-439 SDPA-440 - content types

### PR DESCRIPTION
**Pull request by mugurrus**

### CODE REVIEW

**Comment by petrjasek** [comment]
:+1:

File: server/data/content_types.json [hunk 1]
```diff
@@ -204,5 +204,171 @@
...
+            "abstract": {"required": true},
+            "byline": {},
+            "feature_image": {"type": "picture", "required": true},
+
+            "topic_news": {"required": true},
+            "age_range": {},
+            "territory": {"required": true},
+            "reader_type": {},
+            "assets": {},
+            "relatedItems": {}
```
Comment (gbbr) [inline server/data/content_types.json]: Some of these items in the schema need to have translations on the client ([see here](https://github.com/superdesk/superdesk-client-core/blob/master/scripts/superdesk-workspace/content/content.js#L533-L555)). For example, `relatedItems` and `assets` isn't there at the moment. 

Unfortunately, this is not a very maintainable solution and a better one would be required. I would suggest adding another field in the backend, named `label`, which would describe what the editor should show. Any other suggestions or solutions are welcome.

**Comment by gbbr** [comment]
Besides the missing translations, this LGTM 👍

File: server/data/content_types.json (hunk 1 above)
Comment (mugurrus) [inline server/data/content_types.json]: Thanks, will do

**Pull request merged**